src/main/resources/assets/schematica/lang/de_DE.lang

```diff
@@ -28,23 +28,23 @@ schematica.gui.materials=Materialien
 schematica.gui.materialname=Material
 schematica.gui.materialamount=Menge
 schematica.gui.printer=Drucker
-# schematica.gui.unknownblock=Unknown Block
+# schematica.gui.unknownblock=Unbekannter Block
```

Comment:
You'll want to remove the `#<space>` before the entries where you localized things